docs/community/com-config/old-size-featurebase-database.md

```diff
@@ -13,7 +13,9 @@ grand_parent: Community
 
 ## Determining Hardware Requirements
 
-Ingesters in FeatureBase are stateless and can be deployed in containers and easily scaled up and down. FeatureBase is stateful and has widely varying hardware requirements depending on the size of the data and query workload. FeatureBase can also be scaled up and down, but there's enough overhead in this process that you wouldn't want to be resizing it constantly in response to shifting demand. You may also need to adjust some operating system configuration features to take full advantage of larger systems; see [host system requirements](/docs/community/old-setup/old-hostsystem).
+Ingesters in FeatureBase are stateless and can be deployed in containers and easily scaled up and down. FeatureBase is stateful and has widely varying hardware requirements depending on the size of the data and query workload. FeatureBase can also be scaled up and down, but there's enough overhead in this process that you wouldn't want to be resizing it constantly in response to shifting demand. You may also need to adjust some operating system configuration features to take full advantage of larger systems.
+
+* [Learn how to address performance issues](/docs/community/com-troubleshooting/com-issue-memory-processes)
```

Comment:
Seems out of place. why is this with resizing?

Reply:
Because that's what's in the original (quote from old docs/size-featurebase-database)

```
## Determining Hardware Requirements
Ingesters in FeatureBase are stateless and can be deployed in containers and easily scaled up and down. FeatureBase is stateful and has widely varying hardware requirements depending on the size of the data and query workload. FeatureBase can also be scaled up and down, but there's enough overhead in this process that you wouldn't want to be resizing it constantly in response to shifting demand. You may also need to adjust some operating system configuration features to take full advantage of larger systems; see [host system requirements](/community/community-setup/hostsystem).
```

Reply:
FYI rejigged the links because the com-issue-memory-processes.md file was deleted as requested.
This page is going to be given a big rewrite in next set of changes :)